python/TestHarness/RunParallel.py

```diff
@@ -83,7 +83,8 @@ def run(self, tester, command, recurse=True, slot_check=True):
       if self.soft_limit:
         self.big_queue.append([tester, command, os.getcwd()])
       else:
-        self.harness.handleTestResult(tester, '', 'skipped (Insufficient slots)')
+        tester.setStatus('Insufficient slots', 'SKIP')
+        self.harness.handleTestResult(tester, '', '')
         self.skipped_jobs.add(tester.specs['test_name'])
```

Comment:
I think I found the last one...  Where the TestHarness was sending results to the handler, and we were not setting a bucket status. This one was coming back as PASS, because that is how we now have the tester start... A passing status.

The TestHarness is now chalked full of these. Perhaps I should be setting status while in the handleTestResult method... that way we would not have these two lines littered everywhere...?